examples/connect/dash/README.md

```diff
@@ -0,0 +1,37 @@
+# Dash Example
+
+## Start the app locally
+
+```bash
+python app.py
```

Comment:
I think we should talk about the required environment variables before we show how to run the app. This fails if the user hasn't set `DATABRICKS_HOST` and `DATABRICKS_PATH` in their shell